src/main/java/objectsBasicsMandatoryPracticalTasks/Dogs/Size.java

```diff
@@ -0,0 +1,25 @@
+package objectsBasicsMandatoryPracticalTasks.Dogs;
+
+import java.util.Scanner;
+
+class Size {
+    private String[] arrSize;
+
+    String[] inputSize(boolean dummy) {
+        if (dummy) {
+            String[] dummySizes= {"small", "medium", "big"};
+            return dummySizes;
+        }
+        Scanner s1 = new Scanner(System.in);
+        System.out.print("Specify sizes of dogs: small/medium/big (use space for separation): ");
+        String str = s1.nextLine();
+        String[] arrSize = str.split(" ");
```

Comment:
Please use appropriate names for variables which makes sense when you read code.
I.e. s1 -> scanner, str -> dogSizes, arrSize -> arrayOfSizes